### ISSUE TITLE
Travis build failures due to PHP memory errors during composer install

### ISSUE THREAD

**Issue by shrop** [drupal-composer/drupal-composer-project#561]
Travis builds are failing due to PHP out of memory errors during composer install. This has been resolved in Guardr (same issue) with the following change: https://github.com/guardrdistro/guardr-project/blob/8.x/.travis.yml#L27 See related issue for Guardr: https://www.drupal.org/project/guardr/issues/3094170. Credit to @joestewart for the work around this fix.

See: https://travis-ci.org/github/drupal-composer/drupal-project/jobs/673930113

```
$ composer --verbose install

> pre-update-cmd: DrupalProject\composer\ScriptHandler::checkComposerVersion

Loading composer repositories with package information

Updating dependencies (including require-dev)

PHP Fatal error:  Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///home/travis/.phpenv/versions/7.0.25/bin/composer/src/Composer/DependencyResolver/Solver.php on line 223

Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///home/travis/.phpenv/versions/7.0.25/bin/composer/src/Composer/DependencyResolver/Solver.php on line 223

Check https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors for more info on how to handle out of memory errors.The command "composer --verbose install" failed and exited with 255 during .

Your build has been stopped.
```

**Comment by shrop** [comment]
Relates to #558 because some Travis builds are failing due to out of memory errors while fixing this issue with Travis composer snapshots.

See fix in PR: https://github.com/drupal-composer/drupal-project/pull/560 and commit: https://github.com/drupal-composer/drupal-project/pull/560/commits/f3daa5a444b89137a95143759ea40ce64f781106

**Comment by shrop** [comment]
All builds are passing now per: https://travis-ci.org/github/drupal-composer/drupal-project/builds/673931674.

**Comment by normanlolx** [comment]
Seems to have been fixed in #575
